packages/postgres-database/src/simcore_postgres_database/models/projects.py

```diff
@@ -67,7 +67,6 @@ class ProjectType(enum.Enum):
             onupdate="CASCADE",
             ondelete="RESTRICT",
         ),
-        nullable=True,
```

Comment:
never noticed thisone!. 

Q: since we have project_access_rights, dont you think that the concept of "ownership" is redundant? Imaging A creates the project and gives full access to B. Then B removes all access writes to the project from A. Who is the owner?

Reply:
I have removed it from this PR for now. I will add it in separate PR - I will need to check and modify some test that are failing.